xocto/storage/storage.py

```diff
@@ -1423,6 +1440,12 @@ def get_size_in_bytes(self, *, s3_object: S3Object) -> int:
         assert s3_object.bucket_name == self.bucket_name
         return len(self.fetch_file_contents(s3_object.key))
 
+    def get_last_modified(self, key_path: str) -> datetime.datetime:
+        """
+        Given this is used for testing purposes, it's not as relevant.  Just return current time.
+        """
+        return datetime.datetime.now()
```

Comment:
Inside a truncated PR-style view this doesn't make any sense, even though it is for memory storage. In a future PR maybe we can clarify things?

Reply:
Thanks.  Will do.